test/integration/deployers/helmdeployer/helm_deployer_tests.go

```diff
@@ -0,0 +1,373 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package helmdeployer
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/utils/pointer"
+
+	kutil "github.com/gardener/landscaper/controller-utils/pkg/kubernetes"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	"github.com/gardener/landscaper/pkg/deployer/helm"
+	lsutils "github.com/gardener/landscaper/pkg/utils/landscaper"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Achim has updated ginkgo to ginkgo/v2:

```suggestion
	. "github.com/onsi/ginkgo/v2"
```